components/topbar/user/src/index.js

```diff
@@ -185,28 +185,28 @@ class TopbarUser extends Component {
     const toggleMenuClassName = cx('sui-TopbarUser-toggle', {
       'has-notifications': hasNotifications
     })
-    const {icon: NavCtaIcon, url: navCtaUrl, text: navCtaText} = navCTA
+    const { icon: NavCtaIcon, url: navCtaUrl, text: navCtaText, onClick: navCtaOnClick = () => ({}) } = navCTA
```

Comment:
You can resume it like this `() => {}`

Reply:
ok

Reply:
`() => ({})` returns an empty object, `() => {}` returns undefined.
Nevermind maybe a defaultProp would be more accurate.